core/src/main/kotlin/org/evomaster/core/problem/rest/classifier/inputEncoder.kt

```diff
@@ -0,0 +1,48 @@
+package org.evomaster.core.problem.rest.classifier
+
+import org.evomaster.core.problem.rest.data.RestCallAction
+import org.evomaster.core.search.gene.BooleanGene
+import org.evomaster.core.search.gene.collection.EnumGene
+import org.evomaster.core.search.gene.numeric.DoubleGene
+import org.evomaster.core.search.gene.numeric.IntegerGene
+
+
+// Encoding IntegerGene, DoubleGene, EnumGene, and BooleanGene of a RestCallAction into a double vector
+fun inputEncoder(input: RestCallAction): List<Double> {
```

Comment:
it is possible to have package-level functions in Kotlin, but it is not something we use in EvoMaster. rather create a `object` utility class, with this function inside it